### PR TITLE
MSKINS-107: Added the meta generator entry.

### PR DESCRIPTION
**Pull request by smaarn**
Added the generation of the meta entry of name "generator" mentioning the doxia site renderer being used.

### CODE REVIEW

**Comment by michael-o** [comment]
LGTM, I will triage and merge.

File: doxia-site-renderer/src/main/resources/org/apache/maven/doxia/siterenderer/resources/default-site.vm [hunk 1]
```diff
@@ -17,7 +17,7 @@
 ## under the License.
 ##
 #parse( "default-site-macros.vm" )
-<!-- Generated by Apache Maven Doxia Site Renderer#if( $doxiaSiteRendererVersion ) $doxiaSiteRendererVersion#end##
+<!-- Generated by #generator($doxiaSiteRendererVersion)
```
Comment (michael-o) [inline doxia-site-renderer/src/main/resources/org/apache/maven/doxia/siterenderer/resources/default-site.vm]: This name is inconsistent with apache/maven-fluido-skin#9. Please unify.

Comment (michael-o) [reply doxia-site-renderer/src/main/resources/org/apache/maven/doxia/siterenderer/resources/default-site.vm]: Akin to 57405f71ea3d38840f9711fb9b2c02f5033eb768.

Comment (smaarn) [reply doxia-site-renderer/src/main/resources/org/apache/maven/doxia/siterenderer/resources/default-site.vm]: I unified with the merged result in the apache/maven-fluido-skin master (indeed passing a parameter was redundant, my bad).

**Comment by smaarn** [comment]
Force pushed the alignment on the fluido-skin project.

**Comment by michael-o** [comment]
Please create the corresponding DOXIASITETOOLS ticket.

**Comment by smaarn** [comment]
Corresponding DOXIASITETOOLS ticket created: DOXIASITETOOLS-207.
Closing this PR.

**Pull request closed without merging**